group-community/communication.rst

```diff
@@ -0,0 +1,51 @@
+What problem are we trying to solving?
+======================================
+
+We recognise that although a lot of people know and love IRC it does have some short commings for some people, such as:
+
+* Unless you are running a persistent connection (bouncer) you lose history
+* Not everybody likes IRC
+
+The Ansible Community Working Group is looking at making it easier for people to join the Ansible Discussions
+
+
+What's the proposed solution?
+=============================
+
+We will experiment with using `Matrix.org <https://matrix.org>`_ to allow people to connect to our Freenode channels.
+
+Matrix is an ecosystem for open and interoperable communication.
+
+How can we use Matrix?
+======================
+
+1. Create a `Matrix.org account <https://matrix.org/beta/#/register>`_
+2. Join `ansible on Matrix <https://matrix.to/#/#freenode_#ansible:matrix.org>`_, if you are just starting we suggest riot.im
+  
+   a. Sign into `#ansible on riot.im <https://riot.im/app/#/room/#freenode_#ansible:matrix.org>`_ using the credentials you created in step 1
```

Comment:
If the link above is updated to a Riot link - then you probably won't need to re-sign in